contribs/gnokeykc/main.go

```diff
@@ -17,7 +19,9 @@ func main() {
 	cmd.AddSubCommands(newKcCmd(stdio))
 
 	if err := cmd.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
-		_, _ = fmt.Fprintf(os.Stderr, "%+v\n", err)
+		if !errors.Is(err, flag.ErrHelp) {
+			_, _ = fmt.Fprintf(os.Stderr, "%+v\n", err)
+		}
 
 		os.Exit(1)
 	}
```

Comment:
Since this logic is repeated for every command, I propose you modify tm2/pkg/commands to have a helper function `func (c *Command) Main(args []string)` which essentially wraps this logic here (ParseAndRun with background and args, and exit if the error is not nil).

Reply:
@thehowl Hi sir, suggestion changes have been addressed. Thanks for your review